pkg/parsers/kernel/uname_solaris.go

```diff
@@ -0,0 +1,14 @@
+package kernel
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+func uname() (*unix.Utsname, error) {
+	uts := &unix.Utsname{}
+
+	if err := unix.Uname(uts); err != nil {
```

Comment:
I think it would. The only thing is that on Solaris golang.org/x/sys/unix is delivered internally so vendoring is not required. For it to work for other platforms it would have to be vendored in the engine. Is that ok?